### ISSUE TITLE
grimshot option to hide cursor

### ISSUE THREAD

**Issue by whoizit** [swaywm/sway#5895]
I could not find an option that would hide the cursor in screenshots.
For example with command:
```
set $bindsym bindsym --to-code
set $src /d/screenshot
$bindsym {
...
Shift+Print         exec grimshot save active   $scr/$(date +"%F_%T_grimshot.png")
...
}
```

### Please read the following before submitting:
- We are not accepting any new window management features unless they get implemented by i3. Please consider searching for or opening an i3 feature request.

### Please fill out the following:
- **Description:**
Please describe in plain English what the enhancement is and what the use case is.

**Comment by emersion** [comment]
Sorry, we don't keep track of issues for `contrib/` material. We do accept patches though.
